src/geotop/water.balance.cc

```diff
@@ -1736,7 +1736,6 @@ int find_f_3D(double Dt, Vector<double> *f, ALLDATA *adt, SOIL_STATE *L,
     long i;
     long n=(Nl+1)*adt->P->total_pixel;
 
-    #pragma omp parallel for
```

Comment:
why this commit is also here?

Reply:
because I checked out from a v3.0 without that ```#pragma omp parallel for```

Reply:
yes I figured it out. I say that it is wrong. please rebase on current v3.0

Reply:
you mean, to do 
```
git checkout v3.0
git rebase inline
```
?

Reply:
done